src/soar_sdk/shims/phantom/install_info.py

```diff
@@ -15,3 +15,10 @@ def get_verify_ssl_setting() -> bool:
 
 
 __all__ = ["get_verify_ssl_setting"]
+
+
+def is_soar_available() -> bool:
+    """
+    Returns whether SOAR is available.
+    """
+    return _soar_is_available
```

Comment:
This needs to be included in the `__all__`. I'm also not 100% convinced it belongs in this module, but either way

```suggestion
def is_soar_available() -> bool:
    """
    Returns whether SOAR is available.
    """
    return _soar_is_available

__all__ = ["get_verify_ssl_setting", "is_soar_available"]

```

Reply:
Yeah tbh I wasn't sure exactly where to put it. I initially had it as a helper in logger.py but didn't want to be importing a module and not use it anywhere just for this check